transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpConnectProcessor.java

```diff
@@ -87,12 +87,13 @@ protected void finishConnect(DefaultHttpSession session) {
                 // TODO: strip port if default
                 httpRequest.setHeader(HttpHeaders.HEADER_HOST, resource.getAuthority());
             }
```

Comment:
I think this if statement is NOT required (as HEADER_HOST is set below unconditionally)